backend/controller/user.go

```diff
@@ -197,6 +208,7 @@ func SendRegistrationEmail(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
+	w.WriteHeader(200)
```

Comment:
it already sends 200, but im fine with keeping this since we do it elsewhere just maybe we don't need to be doing it elsewhere